src/Altinn.App.Core/Internal/AccessManagement/IAccessManagementClient.cs

```diff
@@ -0,0 +1,9 @@
+using Altinn.App.Core.Internal.AccessManagement.Models;
+
+namespace Altinn.App.Core.Internal.AccessManagement;
+
+internal interface IAccessManagementClient
+{
+    public Task<DelegationResponse> DelegateRights(DelegationRequest delegation, CancellationToken ct);
+    public Task<DelegationResponse> RevokeRights(DelegationRequest delegation, CancellationToken ct);
```

Comment:
Usually the cancellation token is optional. Not sure if there are good reasons to make it required here. If you are in a process that is transaction like it makes little sense to stop half way on the cancellation token.

```suggestion
    public Task<DelegationResponse> RevokeRights(DelegationRequest delegation, CancellationToken ct = default);
```